components/needs_team/needs_team.jsx

```diff
@@ -50,14 +54,13 @@ export default class NeedsTeam extends React.Component {
                 team: PropTypes.string.isRequired,
             }).isRequired,
         }).isRequired,
+        history: PropTypes.object.isRequired,
     };
 
     constructor(params) {
         super(params);
 
         this.shortcutKeyDown = (e) => this.onShortcutKeyDown(e);
```

Comment:
Will this work the same if this is removed and just make onShortcutKeyDown (below) as arrow function?

Reply:
It does work. Will fix this.